hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/TableSnapshotInputFormatImpl.java

```diff
@@ -107,6 +107,14 @@
   public static final String SNAPSHOT_INPUTFORMAT_ROW_LIMIT_PER_INPUTSPLIT =
       "hbase.TableSnapshotInputFormat.row.limit.per.inputsplit";
 
+  /**
+   * Whether to enable scan metrics on Scan, default to true
+   */
+  public static final String  SNAPSHOT_INPUTFORMAT_SCAN_METRICS_ENABLED =
+    "hbase.TableSnapshotInputFormat.scan_metrics_enabled";
```

Comment:
Should we name it as scan_metrics.enabled ?  That is what normally we do I believe and I could see many such conf names in code base

Reply:
Thanks for the review, let me change it.